graphics/fontman.cpp

```diff
@@ -188,4 +188,23 @@ const Font *FontManager::getFontByUsage(FontUsage usage) const {
 	return 0;
 }
 
+const char *FontManager::getFontNameByUsage(FontUsage usage) {
```

Comment:
Why is this method needed? Isn't just calling `getFontByUsage()->getFontName()` is enough? This duplicates the font selection logic which is not good.